locales/tr.json

```diff
@@ -210,22 +210,22 @@
     },
     "message": {
         "disclaimerAlert": "Japonya'da sadece Moderna ve Pfizer aşıları onaylanmıştır. SinoVac ve diğer aşıları öneren kliniklerden sakının. Bu sitenin içeriği kullanıcıları tarafından üretilmiştir. Bu içeriği yönetmek için elimizden geleni yapıyoruz, fakat yine de hatalar veya doğrulanmamış bilgiler olabilir. <br /><br /> Topluluk tarafından gönderilmiş olup doğru olmayan bilgilerden ve üçüncü web adreslerinden biz sorumlu değiliz. İçerikleri dikkatle inceleyin ve eğer sahte bir bilgi görürseniz lütfen ihbar edin.",
-        "shinYokohamaAlert": "<h2>NEWS</h2><br /><b>Shin-Yokohama</b> vaccine slots available from <b>July 12th</b> for voucher holders. If you live outside Yokohama and wish to register, <a href='https://v-sys.mhlw.go.jp/application/change-region.html' target='_blank'>click here to fill out and print the request form.</a>THEN click the button to book the vaccine!",
-        "shinYokohamaBookVaccine": "Book Vaccine"
+        "shinYokohamaAlert": "<h2>HABERLER</h2><br />Kuponu olanlara <b>Shin-Yokohama</b>'da <b>21 Temmuz</b>'dan itibaren aşı randevuları açıldı. Bölge ikameti gerekmiyor!",
```

Comment:
We'd like to keep the link in this HTML:

` <a href='https://v-sys.mhlw.go.jp/application/change-region.html' target='_blank'>click here to fill out and print the request form.</a>`

Reply:
Hi @ann-kilzer , thanks for your comment! I was looking at the `en.json` of latest `main`, which didn't have the URL: https://github.com/ourjapanlife/findadoc-localization/blob/main/locales/en.json#L213

I can surely add the URL back, but can you confirm which one is correct?

Reply:
Ohh oops, looks like I am out of date.